src/components/tree_view/tree_view.scss

```diff
@@ -24,7 +24,7 @@
 }
 
 .ouiTreeView__node--expanded {
-  max-height: 100vh;
+  max-height: none;
```

Comment:
```suggestion
  max-height: 100vh;
```

Reply:
This code change should not be in this PR.

Reply:
thanks Willie good catch